project-base/storefront/components/Blocks/FreeTransport/FreeTransportRange.tsx

```diff
@@ -0,0 +1,70 @@
+import Trans from 'next-translate/Trans';
+import useTranslation from 'next-translate/useTranslation';
+import { twJoin } from 'tailwind-merge';
+import { useCurrentCart } from 'utils/cart/useCurrentCart';
+import { useFormatPrice } from 'utils/formatting/useFormatPrice';
+
+const barWrapperTwClass = 'relative order-1 h-[4px] vl:order-0';
+const barTwClass = 'absolute top-1/2 left-0 h-[4px] w-full bg-borderAccentLess';
+const barRangeTwClass =
+    'relative h-[4px] rounded-md bg-actionPrimaryBorder transition-width ease-defaultTransition duration-200';
+
+const FreeTransportRange: FC = () => {
+    const testIdentifier = 'blocks-freetransport-range';
+    const { cart } = useCurrentCart();
+    const { t } = useTranslation();
+    const formatPrice = useFormatPrice();
+
+    const shouldDisplayTransportBar = cart?.remainingAmountWithVatForFreeTransport !== null;
+    const totalItemPrice = Number(cart?.totalItemsPrice.priceWithVat);
+    const totalPriceRemaining = Number(cart?.remainingAmountWithVatForFreeTransport);
+    const totalPriceRemainingPercents =
+        totalItemPrice && totalPriceRemaining ? (totalItemPrice / (totalItemPrice + totalPriceRemaining)) * 100 : 0;
+
+    if (cart?.items.length === 0 || !shouldDisplayTransportBar) {
+        return null;
+    }
+
+    const amountFormatted = formatPrice(totalPriceRemaining);
+
+    const freeTransportRangeTwClass = twJoin(
```

Comment:
Could you please move this const to the other classes on row number 11 to maintain consistency?